### DISCUSSION TITLE
Using the app on Nvidia Shield

### DISCUSSION THREAD

**Discussion by Misrabligen** [bttv-android/bttv#709] [General]
Hi all, this app used to work just fine on my Nvidia Shield, but it seems ever since Twitch introduced the different chat settings/views, I cannot navigate to the classic view that shows the chat on the right hand side including the BTTV emotes, which is the entire purpose of the app - does anyone know how to get it to show the classic view? On mobile it's all good, it's really just on the Shield where I can't make it show the default chat view. Thank you for any answers!

**Comment by FoseFx** [comment]
Not familiar with the device, and also not entirely sure what the exact problem is, but generally you can cycle through the chat modes using this button:
![Screenshot_2024-12-28-16-35-45-04_34750238b5877b2a91c912748cb283f8.jpg](https://github.com/user-attachments/assets/9b16ded0-95a2-450f-9b16-10bfb1ed45d6)

**Reply by Misrabligen** [reply]
Hey, thanks, thing is, I can't even access this menu with the remote of the Nvidia Shield - most of the time, I don't even see which element is currently selected. I understand that this is first and foremost an android MOBILE app, so I hoped that maybe another Shield user had figured out how to see the "classic" chat view
